packages/manager/modules/nutanix/src/dashboard/general-info/routing.js

```diff
@@ -3,6 +3,8 @@ export default /* @ngInject */ ($stateProvider) => {
     url: '',
     component: 'nutanixGeneralInfo',
     resolve: {
+      serviceInfo: /* @ngInject */ (NutanixDashboard, serviceName) =>
+        NutanixDashboard.getServiceInfo(serviceName),
```

Comment:
Do you think NutanixDashboard should be renamed to "NutanixService" or just "Nutanix". I dont thik we need to have a separate service only for dashboard.

Reply:
I think it makes sense to make it as `NutanixService`